unicodetools/src/test/java/org/unicode/text/UCD/TestCodeInvariants.java

```diff
@@ -41,12 +44,24 @@ public static void main(String[] args) {
         testGcbInDecompositions(true);
```

Comment:
I would remove the `showAllNfds` parameter from this function and change its code as if it had been false (or replace `showAllNfds` with `VERBOSE`). It seems unnecessary to do verbose printing for thousands of decompositions that pass the test.

Reply:
ok